src/test/java/seedu/address/logic/parser/ListCommandParserTest.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_LANGUAGE_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_TAG_DESC;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.model.tag.Language;
+import seedu.address.model.tag.LanguageContainsKeywordsPredicate;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.TagContainsKeywordsPredicate;
+
+public class ListCommandParserTest {
+
+    private ListCommandParser parser = new ListCommandParser();
+
+
+    @Test
+    public void parse_validArgs_returnsListCommand() {
+        Set<Tag> testTags = new HashSet<>();
+        Set<Language> testLanguages = new HashSet<>();
+
+        ListCommand expectedListCommand =
+                new ListCommand(new TagContainsKeywordsPredicate(testTags),
+                        new LanguageContainsKeywordsPredicate(testLanguages),
+                        false);
+        // empty argument
+        assertParseSuccess(parser, "", expectedListCommand);
+
+        testTags.add(new Tag("friend"));
+
+        testLanguages.add(new Language("python"));
+        System.out.println(testLanguages);
```

Comment:
The test cases is well done! Would it be cleaner if `println` is removed? I believe this is your reference when you are constructing the test case.